modules/oe_multilingual_url_suffix/oe_multilingual_url_suffix.module

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @file
+ * The OpenEuropa Multilingual URL suffix module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\language\Plugin\LanguageNegotiation\LanguageNegotiationUrlFallback;
+use Drupal\oe_multilingual_url_suffix\Plugin\LanguageNegotiation\LanguageNegotiationUrlSuffix;
+
+/**
+ * Implements hook_language_types_info_alter().
+ *
+ * Fixes the suffix-based language negotiation plugin for the URL type of
+ * language. This basically changes the default URL-based language negotiation
+ * plugin set by core (LanguageNegotiationUrl).
```

Comment:
This explanation does not clarify why we need this. This has been a source of confusion for several team members, let's rework / add to this comment from the explanation available [here](Explanation about why we need this is in here: https://github.com/openeuropa/oe_multilingual/pull/77#pullrequestreview-269608595 ).